content/credits/_index.es.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Creditos"
+weight: 200
+---
+
+# Patrocinado por
+
+[![efabless](/images/efabless.png)](https://efabless.com/)
+
+# Equipo
+
+Tiny Tapeout no sería posible sin la ayuda de muchas personas. Nos gustaría agradecer especialmente a:
+
+* Uri Shaked por el desarrollo de [wokwi](https://wokwi.com/) y mucho más
+* [Sylvain Munaut](https://twitter.com/tnt) por ayudar con mejoras en la cadena de escaneo
+* [Mike Thompson](https://www.linkedin.com/in/michael-thompson-0a581a/) por su experticia en verificación
+* [Jix](https://twitter.com/jix_) por soporte de verificación formal
+* [Proppy](https://twitter.com/proppy) por ayudar con Github actions
+* [Maximo Balestrini](https://twitter.com/maxiborga) por todos los increíbles renders y el visor GDS interactivo
+* James Rosenthal por presentar ejemplos con diseño digital
+* Todas las personas que fueron parte de [TinyTapeout 01](/runs/tt01) y ofrecieron su tiempo de manera voluntaria para mejorar la documentación y probar el flujo.
+* El equipo de [YosysHQ](https://www.yosyshq.com/) y todos los otros desarrolladores de herramientas EDA de código abierto
+* [Efabless](https://efabless.com/) por realizar los lanzamientos y proporcionar OpenLane y patrocinio
+* [Tim Ansell y Google](https://www.youtube.com/watch?v=EczW2IWdnOM) por apoyar al movimiento silicio de código abierto
+* Comunidad del curso [Zero to ASIC](https://zerotoasiccourse.com/) por todo tu apoyo
```

Comment:
please add yourselves for the spanish translation!

Reply:
shure!